src/components/atoms/Button/Button.md

```diff
@@ -11,123 +11,95 @@ It comes in a lot of variations to suit different needs.
+    </FlexCol>
+  </FlexRow>
+</FlexContainer>;
 ```
 
-- Large size ( _rarely to be used, but it's here_ )
+- Loading ( _spinner won't show for `styling="link"`_ )
 
 ```tsx
-import { Button } from '@zopauk/react-components';
+import { Button, FlexRow, FlexCol, FlexContainer } from '@zopauk/react-components';
 
-<Button sizing="large">Big smash</Button>;
+<FlexContainer>
+  <FlexRow>
+    <FlexCol xs={6}>
+      <Button loading>Loading</Button>
+    </FlexCol>
+    <FlexCol xs={6}>
+      <Button styling="secondary" loading>
+        Waiting
+      </Button>
+    </FlexCol>
+  </FlexRow>
+</FlexContainer>;
 ```
 
-- Small size ( _mainly used when many actions are available in close proximity_ )
+- With icons
 
 ```tsx
-import { Button } from '@zopauk/react-components';
-
-<Button sizing="small">Little smash</Button>;
-```
-
-- Compact size ( _for when the link style just won't work_ )
-
-```tsx
-import { Button } from '@zopauk/react-components';
-
-<Button sizing="compact">Tiny smash</Button>;
-```
-
-- With an icon on its right side ( _can be used with arrows, among other icons_ )
-
-```tsx
-import { AlertIcon, Button } from '@zopauk/react-components';
-
-<Button rightIcon={<AlertIcon fillColor="#fff" />}>Learn more</Button>;
-```
-
-- With an icon on its left side ( _for when the icon conveys the meaning faster than text_ )
-
-```tsx
-import { AlertIcon, Button } from '@zopauk/react-components';
-
-<Button leftIcon={<AlertIcon fillColor="#fff" />}>Smash now</Button>;
-```
-
-- Full width ( _for when the button should stretch the entire width of the divider_ )
-
-```tsx
-import { Button } from '@zopauk/react-components';
-
-<Button fullWidth={true}>Button smash</Button>;
-```
-
-- Primary contrast ( _note that the text colour is the same as the background_ )
-
-```tsx { "props": { "style": { "backgroundColor": "#141E64", "border": "2px solid #efefef" } } }
-import { Button, colors } from '@zopauk/react-components';
-
-<Button styling="contrastPrimary" contrastColor={colors.greyDark}>
-  Button smash
-</Button>;
-```
-
-- Secondary contrast ( _the border should be the colour's shade in 50 or close to that_ )
-
-```tsx { "props": { "style": { "backgroundColor": "#141E64", "border": "2px solid #efefef" } } }
-import { Button, colors } from '@zopauk/react-components';
-
-<Button styling="contrastSecondary" contrastColor={colors.greyLighter}>
-  Button smash
-</Button>;
-```
-
-- Text button contrast ( _see above_ )
-
-```tsx { "props": { "style": { "backgroundColor": "#141E64", "border": "2px solid #efefef" } } }
-import { Button } from '@zopauk/react-components';
-
-<Button styling="contrastLink">Link smash</Button>;
+import { Button, FlexRow, FlexCol, FlexContainer } from '@zopauk/react-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowCircleRight, faArrowCircleLeft, faCoffee } from '@fortawesome/free-solid-svg-icons';
+
+<FlexContainer>
+  <FlexRow>
+    <FlexCol xs={4}>
+      <Button>
```

Comment:
[Opinion] It will look better if we add a fullWidth to all the buttons!

Reply:
Unfortunately, that would have the potential to cause a lot of layout issues given the number of apps this library integrates with 📐

Layout issues are the hardest to track given rarely there's tests covering them 😓 so I prefer to walk the safe road and keep the existing behaviour.

Reply:
I meant in this documentation file! hahah. sorry